src/components/Landing/EventCard/EventCard.jsx

```diff
@@ -1,73 +1,75 @@
-import React, { useState, useEffect, useMemo } from "react";
+import React, { useState, useEffect } from "react";
 import { NavLink } from "react-router-dom";
 import PropTypes from "prop-types";
 import { BiRightArrowAlt } from "react-icons/bi";
 import moment from "moment";
-import IconsEventCard from "./IconsEventCard/IconsEventCard";
 import "./EventCard.scss";
 import PhotoService from "../../../services/PhotoService";
```

Comment:
I sad that we need to remove `PhotoService` from component

Reply:
deleted